/*
 * Created By: Gareth Ellis
 * Created On: January 9, 2018
 * Description: TODO
 */

#ifndef MAPPING_IGVC_OBSTACLEMANAGER_H
#define MAPPING_IGVC_OBSTACLEMANAGER_H

#include <mapping_igvc_msgs/ConeObstacle.h>

class ObstacleManager {
    // Add obstacle (should have versions for polynomial line and cone)
};


#endif //MAPPING_IGVC_OBSTACLEMANAGER_H
